hathor/p2p/manager.py

```diff
@@ -158,6 +158,16 @@ def stop(self) -> None:
         if self.lc_reconnect.running:
             self.lc_reconnect.stop()
 
+    def _get_peers_count(self) -> Dict[str, int]:
+        """Get a dict containing the count of peers in each state"""
+
+        return {
```

Comment:
I don't like using dicts internally because it messes up with linters. For instance, a linter cannot check whether a given key exists or not. I feel we should use named tuples instead here.

Reply:
Fixed in https://github.com/HathorNetwork/hathor-core/pull/464/commits/3c0ab8048a073a89664b618bee4043f0599cef9f

Reply:
Thanks!